src/components/table-field.pcss

```diff
@@ -22,15 +22,23 @@
   }
 
   &.all-excluded {
-    background-color: #e8e8e8;
+    opacity: 0.5;
   }
 
   &.single-hit {
-    box-shadow: inset 0 0 4px #ff0;
+    .bartext {
+      color: #f29600;
+      font-weight: 600;
+      font-size: 1.5em;
+    }
   }
 
   &.filtered {
-    box-shadow: inset 0 0 2px #0f0;
+    .name {
+      background-color: #019b39;
+      padding: 0 .25em;
```

Comment:
I would like a bit more left/right padding

Reply:
.5em, thats all I can give you ;)

Reply:
I would have settled on .4